x-pack/plugins/endpoint/public/applications/endpoint/store/policy_list/index.test.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { PolicyListState } from '../../types';
+import { applyMiddleware, createStore, Dispatch, Store } from 'redux';
+import { AppAction } from '../action';
+import { policyListReducer } from './reducer';
+import { policyListMiddlewareFactory } from './middleware';
+import { coreMock } from '../../../../../../../../src/core/public/mocks';
+import { CoreStart } from 'kibana/public';
+import { selectIsLoading } from './selectors';
+
+describe('policy list store concerns', () => {
+  const sleep = () => new Promise(resolve => setTimeout(resolve, 100));
+  let fakeCoreStart: jest.Mocked<CoreStart>;
+  let store: Store<PolicyListState>;
+  let getState: typeof store['getState'];
+  let dispatch: Dispatch<AppAction>;
+
+  beforeEach(() => {
+    fakeCoreStart = coreMock.createStart({ basePath: '/mock' });
+    store = createStore(
+      policyListReducer,
+      applyMiddleware(policyListMiddlewareFactory(fakeCoreStart))
+    );
+    getState = store.getState;
+    dispatch = store.dispatch;
+  });
+
+  test('it sets `isLoading` when `userNavigatedToPage`', async () => {
+    expect(selectIsLoading(getState())).toBe(false);
+    dispatch({ type: 'userNavigatedToPage', payload: 'policyListPage' });
+    expect(selectIsLoading(getState())).toBe(true);
+    await sleep();
```

Comment:
does this cause the mock core start to return response?

Reply:
I don't actually make the API call at this time in the middleware, so not yet.
But to your question: yes, you can setup the mocked interface to return a specific response.  Here is an example in Candace's PR - https://github.com/elastic/kibana/pull/55623/files#diff-a4d4b2d400e6092b6e6a5ca7a6d9a8eeR69